Delegate release notes - 791xx (#1397)
harness/developer-hub: release-notes/delegate.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest release - April 21, 2023, Harness version 791xx, Harness Delegate version 79106

Harness NextGen release 791xx includes the following changes for the Harness Delegate.

```mdx-code-block
<Tabs>
  <TabItem value="What's new">
```
### What's new

This release introduces the following new features and enhancements:

> - Added the following metrics for immutable delegates that you can scrape via prometheus: (DEL-5363)

Reviewer: ```suggestion
- Added the following metrics for immutable delegates that you can scrape via Prometheus: (DEL-5363)
```

> - Added the following metrics for immutable delegates that you can scrapge via prometheus: (DEL-5363)

Reviewer 2: ```suggestion
- Added the following metrics for immutable delegates that you can scrape via prometheus: (DEL-5363)
```

### Fixed issues

This release includes the following fixes:

> - Added WebSocket reconnect logic when Harness Manager does not recieve a heartbeat handshake from Harness Delegate for more than five minutes. (DEL-5954)

Reviewer: ```suggestion
- Added WebSocket reconnect logic for when the Harness Manager does not receive a heartbeat from the Harness Delegate for more than five minutes. (DEL-5954)
```

Reviewer: The ticket summaries we get are often not the best in terms of grammar. Always revise them for clarity, grammar, etc.

> - Set the `LANG` environment variable to en_US.UTF-8 by default for delegates. (DEL-6221)

Reviewer: ```suggestion
- Set the delegate `LANG` environment variable to en_US.UTF-8 by default. (DEL-6221)
```

<details>
<summary>Expand this section to view changes to previous releases</summary>

#### March 31, 2023, Harness version 78914, Harness Delegate version 78904

> Harness NextGen release 78914 includes the following changes for the Harness Delegate.

Reviewer: ```suggestion
Harness release 78914 includes the following changes for the Harness Delegate.
```

Reviewer: you don't need to say NextGen since these are the NG release notes.

##### What's new

> - Added support for the latest Git CLI in the delegate maximal image. (DEL-6121)
> 
> - The latest Git CLI is now included by default.

Reviewer: ```suggestion
- Added support for the latest Git CLI in the delegate maximal image. (DEL-6121)
  - The latest Git CLI is now included by default.
```

Reviewer: I think the second bullet is subordinate, right?
